src/library/Form/FormField.js

```diff
@@ -99,7 +99,7 @@ export default class FormField extends Component<FormFieldProps> {
             <span {...labelTextProps}>{label}</span>
             {(required || secondaryText) && (
               <FormFieldSecondaryText secondaryText={secondaryText}>
-                {secondaryText ? secondaryText : requiredText}
+                {secondaryText ? secondaryText : requiredText ? requiredText : null}
```

Comment:
@tx-steven  I just realized -- isn't this line just the same as `{secondaryText || requiredText || null}` which might be a bit more readable.